pulp_python/app/models.py

```diff
@@ -79,6 +79,11 @@ class PythonPackageContent(Content):
     obsoletes_dist = models.TextField(default="[]", blank=False)
     requires_external = models.TextField(default="[]", blank=False)
 
+    class Meta:
+        unique_together = (
```

Comment:
This was added because core either uses a "natural_key" or creates one using "unique_together"